test/battle/move_effect/paralyze_hit.c

```diff
@@ -21,7 +21,7 @@ SINGLE_BATTLE_TEST("Thunder Shock inflicts paralysis")
     }
 }
 
-SINGLE_BATTLE_TEST("Thunder Shock cannot paralyze an Electric-type")
+SINGLE_BATTLE_TEST("Paralysis cannot be inflicted on Electric-type Pokémon")
```

Comment:
This test should only check the `EFFECT_PARALYZE_HIT`, not the general Electric-type immunity.
```suggestion
SINGLE_BATTLE_TEST("Thunder Shock cannot paralyze an Electric-type")
```